tests/phpunit/tests/test-accept-languages-collection.php

```diff
@@ -122,25 +142,29 @@ public function test_pick_matching_language_and_variant_when_region_is_missing()
 
 		$best_match = $accept_languages->find_best_match( $languages );
 
-		$this->assertEquals( $de_de_formal->slug, $best_match );
+		$this->assertSame( $de_de_formal->slug, $best_match );
 	}
 
 	public function test_pick_matching_language_and_region_with_custom_slug() {
-		$accept_languages = PLL_Accept_Languages_Collection::from_accept_language_header( 'zh-HK' );
-		$zh_cn = new PLL_Language(
-			array_merge(
-				self::$known_languages['zh_CN'],
-				array(
-					'slug' => 'zh-cn',
-					'w3c'  => 'zh-CN', // Is computed from locale when language is set from term. {@see PLL_Language::__construct()}.
-				)
-			)
-		);
-		$languages = array( $zh_cn );
+		$accept_languages    = PLL_Accept_Languages_Collection::from_accept_language_header( 'zh-HK' );
+		$zh_cn               = self::$known_languages['zh_CN'];
+		$zh_cn['locale']     = 'zh_CN';
+		$zh_cn['slug']       = 'zh-cn'; // Custom slug.
+		$zh_cn['w3c']        = 'zh-CN';
+		$zh_cn['rtl']        = 'rtl' === $zh_cn['dir'] ? 1 : 0;
+		$zh_cn['term_group'] = 0;
+		$result              = self::$model->add_language( $zh_cn );
```

Comment:
Same as above. The method [is able to handle custom data](https://github.com/polylang/polylang/blob/3.3/tests/phpunit/includes/testcase-trait.php#L95).

Reply:
Done in 2df8e11